### PR TITLE
bumped massa-web3 and made some extensions

### PR DESCRIPTION
**Pull request by BatiGencho**

### CODE REVIEW

File: packages/sc-deployer/src/index.ts [hunk 1]
```diff
@@ -23,14 +24,19 @@ const __filename = fileURLToPath(import.meta.url);
...
+
 async function deploySC(
   publicApi: string,
   account: IAccount,
   contracts: ISCData[],
   fee: number = 0,
   maxGas: number = 1_000_000,
   wait: boolean = false,
-) {
+): Promise<IDeploymentInfo> {
```
Comment (Thykof) [inline packages/sc-deployer/src/index.ts:39]: why does it say it returns a promise whereas the return is a js object?

**Review by Thykof — APPROVED** [review]

File: packages/sc-deployer/src/index.ts (hunk 1 above)
Comment (peterjah) [reply packages/sc-deployer/src/index.ts:39]: async function always returns a Promise

Comment (peterjah) [reply packages/sc-deployer/src/index.ts:39]: And using `await` on it will unwrap the object from the Promise

**Review by peterjah — APPROVED** [review]

File: packages/sc-deployer/src/index.ts (hunk 1 above)
Comment (BatiGencho) [reply packages/sc-deployer/src/index.ts:39]: correct!

Comment (Thykof) [reply packages/sc-deployer/src/index.ts:39]: awesome thanks!

**Pull request merged**